react/LocationCountry.tsx

```diff
@@ -116,7 +116,6 @@ const LocationCountry: React.FC<LocationCountryProps> = ({
       <ListboxInput
         translate={undefined}
         aria-labelledby={labelId}
-        className="h-100"
```

Comment:
For some reason, I added this a while ago, but now it's breaking on cart shipping

![image](https://user-images.githubusercontent.com/26108090/103227377-fdb70f80-490c-11eb-99d9-f3bd6f8babc2.png)

After undoing this change, the component seems fine on cart and checkout 🤷🏾

Reply:
nice :satisfaction:

Reply:
The delivery options in cart in Safari is broken:

<img width="410" alt="image" src="https://user-images.githubusercontent.com/11471014/103306392-35e14f80-49ec-11eb-916a-241c5b4f2409.png">

I don't know if it's this PR's scope (since the whole shipping step in checkout is broken too), but it worth checking.

Reply:
I'll take a look

Reply:
It's the device coordinates component again :-/
We should definitely open a task for that, but this isn't related to this PR indeed